core/src/replay_stage.rs

```diff
@@ -3263,32 +3263,39 @@ impl ReplayStage {
                     let ComputedBankState {
                         voted_stakes,
                         total_stake,
+                        bank_stake,
+                        bank_weight,
                         lockout_intervals,
                         my_latest_landed_vote,
                         ..
                     } = computed_bank_state;
                     let stats = progress
                         .get_fork_stats_mut(bank_slot)
                         .expect("All frozen banks must exist in the Progress map");
+                    stats.bank_stake = bank_stake;
                     stats.total_stake = total_stake;
                     stats.voted_stakes = voted_stakes;
                     stats.lockout_intervals = lockout_intervals;
+                    stats.fork_weight = bank_weight;
                     stats.block_height = bank.block_height();
                     stats.my_latest_landed_vote = my_latest_landed_vote;
                     stats.computed = true;
                     new_stats.push(bank_slot);
                     datapoint_info!(
                         "bank_weight",
                         ("slot", bank_slot, i64),
+                        ("bank_stake", stats.bank_stake, i64),
                         // u128 too large for influx, convert to hex
-                        ("weight", format!("{:X}", stats.weight), String),
+                        ("weight", format!("{:X}", stats.fork_weight), String),
                     );
+
                     info!(
-                        "{} slot_weight: {} {} {} {}",
+                        "{} slot_weight: {} {:2}% {}% {:.2} {}",
                         my_vote_pubkey,
                         bank_slot,
-                        stats.weight,
-                        stats.fork_weight,
+                        100 * stats.bank_stake / stats.total_stake, // percentage vote stake in total_stake
+                        100 * stats.fork_weight / stats.total_stake as u128,
```

Comment:
this line and the one below it can be removed

Reply:
yeah.